daprdocs/content/en/reference/components-reference/supported-configuration-stores/azure-appconfig-configuration-store.md

```diff
@@ -88,6 +88,18 @@ You need an Azure subscription to set up Azure App Configuration.
    In a production-grade application, follow [the secret management]({{< ref component-secrets.md >}}) instructions to securely manage your secrets.
    {{% /alert %}}
 
+## Azure App Configuration Request Metadata 
+
+The Azure App Configuration store supports the following optional metadata property:
+
+`label`: The label of the configuration to retrieve. If not present, the configuration store will return the configuration for the specified key and null label.
```

Comment:
It would be good to add 
```suggestion
`label`: The label of the configuration to retrieve. If not present, the configuration store returns the configuration for the specified key and a null label.
```